### PR TITLE
add Huaweicloud as supported list of clouds for testbed.

### PR DESCRIPTION
**Pull request by swaroopar**
Hi, we tried to install the test bed on Huaweicloud EU region and it worked. So updating the documentation to reflect the same. 
Thank you.

### CODE REVIEW

**Comment by swaroopar** [comment]
Hi Team, Could you please see if there is a possibility to restart the failing CI job? the failure does not seem to be coming from the changes in this PR. Thanks in advance.

**Comment by swaroopar** [comment]
Hi Team, the job failed again. Can you please start it again?

File: doc/requirements.md [hunk 1]
```diff
@@ -203,4 +203,32 @@ export ENVIRONMENT=betacloud
 >     auth_type: "v3applicationcredential"
 > ```
 
+* [HuaweiCloud](https://www.huaweicloud.com/eu/)
+
+:::note
+
+>**note:**
+>
+> * Regsistration is possible via https://www.huaweicloud.com/eu/
```
Comment (osfrickler) [inline doc/requirements.md:212]: Registration

**Review by osfrickler — CHANGES_REQUESTED** [review]
Thanks for your contribution, sadly the CI is a bit unstable currently. The failures are not related to your patch, so you can ignore them. Please see my comments below.

File: terraform/environments/huaweicloud.tfvars [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+# customisation:neutron_floatingip
+# override:manager_boot_from_volume
+# override:neutron_router_enable_snat
+# override:nodes_boot_from_volume
+availability_zone         = "eu-west-101b"
+volume_availability_zone  = "eu-west-101b"
+network_availability_zone = "eu-west-101b"
+flavor_node               = "c6s.4xlarge.2"
+flavor_manager            = "s6.2xlarge.2"
+image                     = "Ubuntu 20.04 server 64bit"
```
[the remark this answers is not in the document]
Comment (swaroopar) [reply terraform/environments/huaweicloud.tfvars]: updated.

**Review by osfrickler — APPROVED** [review]
thx, CI has some unrelated failures again, ignoring those

**Pull request merged**